packages/google_maps_flutter/google_maps_flutter_platform_interface/test/types/point_of_interest_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('$PointOfInterest', () {
```

Comment:
No `$`; that pattern is slowly being removed everywhere as it interferes with IDE test UI.

Reply:
I have removed it.